### PR TITLE
rewrite wrote-about-us section with sass, BEM

### PR DESCRIPTION
**Pull request by igorjv123**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -200,14 +201,12 @@ <h1 class="review-header text-center w-100 text-light">What people say about us?
...
-            <div class="container-fluid">
-                <div class="row d-flex justify-content-around mr-1 ml-1 line-after">
-                    <img class="d-block col-md-2 col-sm-6 col-xl-2 col-lg-2 col-6 wrote-about-us_img" src="./img/Forbes_logo.svg.png" alt="">
-                    <img class="d-block col-md-2 col-sm-6 col-xl-2 col-lg-2 col-6 wrote-about-us_img" src="./img/the-new-york-times-logo-vert.png" alt="">
-                    <img class="d-block col-md-2 col-sm-6 col-xl-2 col-lg-2 col-6 wrote-about-us_img" src="./img/2000px-The_Wall_Street_Journal_Logo.svg.png" alt="">
-                    <img class="d-block col-md-2 col-sm-6 col-xl-2 col-lg-2 col-6 wrote-about-us_img" src="./img/Mashable-Logo.png" alt="">
-                </div>
+            <h1 class="wrote-about-us__title">Wrote about us</h1>
+            <div class="wrote-about-us__images-container">
+                <img class="wrote-about-us__images-container__img" src="./img/Forbes_logo.svg.png" alt="">
```
Comment (LenaAviles) [inline index.html]: you should not create elements of elements in BEM. Please simplify this class to `wrote-about-us__img`.

File: sass/wroteAboutUs.sass [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.wrote-about-us__images-container
```
Comment (LenaAviles) [inline sass/wroteAboutUs.sass]: Why you don't use sass syntax?   
```
.wrote-about-us
    &__images-container
       //styles
```

**Comment by LenaAviles** [comment]
In fact, I expected to see a more complex component to practice the BEM methodology.

File: index.html [hunk 3]
```diff
@@ -200,17 +201,15 @@ <h1 class="review-header text-center w-100 text-light">What people say about us?
...
+            <h1 class="wrote-about-us__title">Wrote about us</h1>
+            <div class="wrote-about-us__images-container">
+                <img class="wrote-about-us__img" src="./img/Forbes_logo.svg.png" alt="">
+                <img class="wrote-about-us__img" src="./img/the-new-york-times-logo-vert.png" alt="">
+                <img class="wrote-about-us__img" src="./img/2000px-The_Wall_Street_Journal_Logo.svg.png" alt="">
+                <img class="wrote-about-us__img" src="./img/Mashable-Logo.png" alt="">
             </div>
         </section>
-        <footer class="container-fluid">
+        <footer class="container-fsluid">
```
Comment (LenaAviles) [inline index.html]: is it right class name?

File: sass/wroteAboutUs.sass [hunk 4]
```diff
@@ -0,0 +1,15 @@
+.wrote-about-us
+    .wrote-about-us__images-container
```
Comment (LenaAviles) [inline sass/wroteAboutUs.sass]: use &!! 
see my previous comment carefully

Comment (igorjv123) [reply sass/wroteAboutUs.sass]: sorry for inattentiveness

**Pull request merged**